lib/matplotlib/style/core.py

```diff
@@ -84,6 +85,73 @@ def use(style):
                 raise IOError(msg % style)
 
 
+def get_substyles(styles):
```

Comment:
Alright, I understand your logic and agree with you.
Considering it, that would mean that in my example, parameters passed in the `main` stylesheet would have a higher priority than the ones in the `a`, `b`, `c`, would it not? Any stylesheet you call from an other one could have any parameter overridden? This is just to be sure I understand properly.
I will create a new PR for this, as soon as I have it working properly